### ISSUE TITLE
Improve accessibility (a11y) of the GeSHi block

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-css#39]
I was using the GeSHi block to document how to fix accessibility (a11y) issues in WordPress and discovered that the default styling on s.b/hm caused the geshi block to fail with contrast errors.

<img width="935" height="155" alt="Image" src="https://github.com/user-attachments/assets/98db834b-eb82-42eb-83be-62ab2cf4c033" />

### Requirements

- 0 contrast errors using WAVE and default styling against the default GeSHi block
- For each supported language type

**Comment by bobbingwide** [comment]
I'd already prototyped a solution that resolved the Color Constrast errors for the HTML
Then I found a couple more Color Contrast problems with a badly coded `[bw_css .]` shortcode, which was inside a Classic block.
I needed to update the logic that changes each failing foreground colour to one that passes WCAG AA.

**Comment by bobbingwide** [comment]
After testing a change to oik to support `[bw_pages format="J"]` and `[bw_pages format="K"]` I discovered a problem with WAVE reporting errors when the output is displayed as the post excerpt.
This problem can be resolved by removing the `wpautop` hook attached to `the_excerpt`.

**Comment by bobbingwide** [comment]
There were also problems with Low Color Contrast on GeSHi output for CSS.
